wagtail/admin/templates/wagtailadmin/home/workflow_pages_to_moderate.html

```diff
@@ -63,6 +65,9 @@ <h2 class="title-wrapper">{% trans 'Awaiting your review' %}</h2>
                                     </span>
                                 {% endfor %}
                             </td>
+                            <td valign="top">
+                                {{ revision.user }}
```

Comment:
can you that this doesn't add an n+1 query?
I have a suspicion `WorkflowPagesToModeratePanel` needs `page_revision__user` added to the queryser `select_related()` declaration

Reply:
You are right!